graphql2/schema.graphql

```diff
@@ -1233,8 +1233,13 @@ input CreateUserNotificationRuleInput {
 
 input UpdateUserContactMethodInput {
   id: ID!
+  type: ContactMethodType
+
+  dest: DestinationInput @experimental(flagName: "dest-types")
```

Comment:
it is never valid to update the destination of a contact method, only the name, and status update preference

`value` is deprecated for that reason, as a new "destination" requires another consent/validation flow.

`type` should also have the deprecation warning, not sure why that was missed.